src/main/resources/assets/computercraft/lua/rom/programs/which

```diff
@@ -0,0 +1,13 @@
+--Made by Wilma456
```

Comment:
@Restioson yes, but that doesn't get included in the compiled version of the mod. Although @dan200  [idea](https://github.com/dan200/ComputerCraft/pull/215#issuecomment-302019291) works too.